### ISSUE TITLE
🐛 Rip DVD to ISO on 2.6.67

### ISSUE THREAD

**Issue by joelang1699** [automatic-ripping-machine/automatic-ripping-machine#1107]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I have checked the wiki

- [X] I have searched the wiki for this problem

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

Hello Guys

About two years ago I was using ARM in a VM to backup DVDs to ISO files which worked wonderfully, until I deleted the VM by accident that is. I'm using ARM (2.6.67) in a docker now, yet MakeMKV will only now rip to MKV, surely I can set this to rip to ISO as before?

Additionally, I installed MakeMKV standalone in docker on the same machine which can rip the same disks straight to ISO. I have a Handbrake client on another system i'll be using to transcode the ISOs to MKV/MP4 so I don't want the built in Handbrake being used.

I'm obviously missing something here?

### To Reproduce

selecting dvd_backup fails

### What OS are you seeing the problem on?

Other (please state in Anything else? section)

### ARM Release Version or Git hash (You can find this in the arm settings page)

v2.6.67

### Expected behavior?

Rip DVD to ISO format only

### Anything else?

Container on Unraid

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by microtechno9000** [comment]
Can't say I have tried to create an ISO of late.
Looking at the code ARM supports backup of a disk through setting RIPMETHOD to either 'backup' or 'backup_dvd'. However, it this will only run when the disk type is Bluray, and doesn't create an ISO.

It seems that since you last had ARM setup, the code has been updated to only create ISO images of data disks. ARM checks if a disk is a 'data disk', and if so, then creates a copy using dd.

The devs are working on rebuilding a lot of the code at the moment as it is a nice patchwork of new features added over the years, and has a lot of issues. Allowing for data/ISO disks will be supported in v3.0, but that is a way off.

tldr: ARM doesn't currently support creating an ISO from any disk other than a data disk.

**Comment by MrJay84** [comment]
Okay mate thanks for the response. At least I know now it wasn't my settings.

I'll have a go at running an older version in a container.

The old ISO method was lighting fast, 10 mins Vs 30 plus for the Mkv method. I have a good many disks to get through.

**Comment by joelang1699** [comment]
Okay mate thanks for the response. At least I know now it wasn't my settings.

I'll have a go at running an older version in a container.

The old ISO method was lighting fast, 10 mins Vs 30 plus for the Mkv method. I have a good many disks to get through.

**Comment by joelang1699** [comment]
Is the Debian install script for v2.5.3 Stable still available anywhere?

I have stacks of DVDs I really need to rip to ISO.

**Comment by SylvainMT** [comment]
> Is the Debian install script for v2.5.3 Stable still available anywhere?
> 
> I have stacks of DVDs I really need to rip to ISO.

You can download the source code for v2.5.3 here: https://github.com/automatic-ripping-machine/automatic-ripping-machine/releases/tag/v2.5.3

You'll likely have to adapt the install script to work for you, as by default it downloads the latest version and then the installation will fail.  Note that you are likely going to need to run Debian 10 or 11 for this.

Good Luck
